src/communications/destination_email.py

```diff
@@ -56,6 +56,7 @@ def __init__(self, params_dict):
         self.auth_enabled = read_bool_from_config('auth_enabled', params_dict)
         self.login = params_dict.get('login')
         self.tls = read_bool_from_config('tls', params_dict)
+        self.include_files = read_bool_from_config('include_files', params_dict, default=True)
```

Comment:
can we use name "attach_files" instead, please?

Reply:
sure, i agree that fits better